liquibase-core/src/main/java/liquibase/command/CommandArgumentDefinition.java

```diff
@@ -146,6 +150,28 @@ public String toString() {
         return returnString;
     }
 
+    /**
+     * Interactively prompt in the CLI for a user to supply a value for this command argument.
+     * @param previouslyPromptedValues the values already provided by the user for other arguments in this interactive
+     *                                 prompting session
+     * @return the value provided by the user, or null if the shouldPrompt method of {@link InteractivePromptableCustomizationWrapper}
+     * signifies that this prompt should not occur
+     */
+    public DataType interactivelyPrompt(Map<String, Object> previouslyPromptedValues) {
```

Comment:
Why are we having to pass the previouslyPromptedValues here?

Reply:
Good question.

This is a carry over from the prompting logic that was born out of the need in `checks customize`. We needed a way to control prompts based on the "answers" that the user provided in previous prompts. Hence, previously prompted values.

Example that applies to `init project`: if the user answers "s" (indicating "skip") to the question of "what filename should I use for the changelog", we need to know to skip a later prompt that asks what type of changelog they want. This implementation worked well in the `checks customize` flow, but feels a bit hamfisted here. I was intending to revisit this in a future story where I actually implement this "skip later prompts based on prior prompts logic".